README.md

```diff
@@ -1,23 +1,27 @@
 # Class Profile Web
+
 =========================
 
 ## About
-The Software Engineering Class Profile is a student-initiative focused on showcasing what the program was like for students. It covers content covering co-op, background, academics, lifestyle, future plans, and answers questions such as “Does it matter if I started coding later than everyone else?” or “How much debt do students graduate with?”.
 
-You can view the class profile at [https://classprofile.andyzhang.net](https://classprofile.andyzhang.net).
+The Software Engineering Class Profile is a student-initiative focused on showcasing what the program was like for students. It covers content covering co-op, background, academics, lifestyle, future plans, and answers questions such as "Does it matter if I started coding later than everyone else?" or "How much debt do students graduate with?".
```

Comment:
We should remove `""Does it matter if I started coding later than everyone else?"` as we actually don't have any graph answering this question anymore

Reply:
switched it for another question :>